languages/php/menhir/ast_php_build.ml

```diff
@@ -682,6 +686,7 @@ and lambda_def env (l_use, ld) =
           comma_list xs
           |> List_.map (function LexicalVar (is_ref, name) ->
                  (is_ref <> None, dname name)));
+                 A.f_doc_comment = None; (* Explictly no doc-string on lambda*)
```

Comment:
Shouldn't we just propagate what we get from the CST? Lambdas won't have doc-comments now but who knows if that could change?